plugins/tracker-resources/src/components/projects/NewProject.svelte

```diff
@@ -0,0 +1,117 @@
+<!--
+// Copyright © 2020 Anticrm Platform Contributors.
+// 
+// Licensed under the Eclipse Public License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may
+// obtain a copy of the License at https://www.eclipse.org/legal/epl-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+-->
+
+<script lang="ts">
+  import { Data, generateId, Ref } from '@anticrm/core'
+  import { DatePresenter, EditBox } from '@anticrm/ui'
+  import {  Card, getClient, UserBox, UserBoxList } from '@anticrm/presentation'
+  import {  IntlString } from '@anticrm/platform'
+  import contact from '@anticrm/contact'
+  import { Project, ProjectStatus, Team } from '@anticrm/tracker'
+  import { createEventDispatcher } from 'svelte'
+  import plugin from '../../plugin'
+  import ProjectStatusSelector from './ProjectStatusSelector.svelte'
+
+  export let space: Ref<Team>
+  const dispatch = createEventDispatcher()
+  const client = getClient()
+
+  const object: Data<Project> = {
+    label: '' as IntlString,
+    description: '',
+    status: ProjectStatus.Planned,
+    lead: null,
+    members: [],
+    comments: 0,
+    attachments: 0,
+    startDate: null,
+    targetDate: null,
+    documents: 0
+  }
+
+  async function onSave(){
+    await client.createDoc(plugin.class.Project, space, object, generateId())
```

Comment:
generateId is not required, it will be done automatically.

Reply:
Done